### PR TITLE
Webhook config cleanup

### PR DESCRIPTION
**Pull request by ykaliuta**
- rename serving certificates secret
- remove MutatingWebhookConfiguration references from kustomization
- remove autogenerated misleading labels

## Merge criteria

- [ ] You have read the [contributors guide](https://github.com/opendatahub-io/opendatahub-operator/blob/incubation/CONTRIBUTING.md).
- [ ] Commit messages are meaningful - have a clear and concise summary and detailed explanation of what was changed and why.
- [ ] Pull Request contains a description of the solution, a link to the JIRA issue, and to any dependent or related Pull Request.
- [ ] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [ ] The developer has manually tested the changes and verified that the changes work

### CODE REVIEW

File: bundle/manifests/opendatahub-operator-webhook-service_v1_service.yaml [hunk 1]
```diff
@@ -3,14 +3,11 @@ kind: Service
 metadata:
   annotations:
     service.beta.openshift.io/inject-cabundle: "true"
-    service.beta.openshift.io/serving-cert-secret-name: opendatahub-operator-controller-webhook-cert
+    service.beta.openshift.io/serving-cert-secret-name: opendatahub-operator-webhook-cert
```
Comment (VaishnaviHire) [inline bundle/manifests/opendatahub-operator-webhook-service_v1_service.yaml:6]: @ykaliuta Can you add why this rename is required?

Comment (ykaliuta) [reply bundle/manifests/opendatahub-operator-webhook-service_v1_service.yaml:6]: > @ykaliuta Can you add why this rename is required?

I have the same question :) https://github.com/opendatahub-io/opendatahub-operator/pull/1190#discussion_r1726897416

Comment (ykaliuta) [reply bundle/manifests/opendatahub-operator-webhook-service_v1_service.yaml:6]: ;) No problem

**Review by zdtsw — APPROVED** [review]

**Review by zdtsw — APPROVED** [review]

**Comment by ykaliuta** [comment]
> put a "onhold" just give @VaishnaviHire more time to check

Looks like no objections?

**Pull request merged**